libnmstate/ifaces/macvtap.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+from libnmstate.schema import MacVtap
+
+from .macvlan import MacVlanIface
+
+
+class MacVtapIface(MacVlanIface):
+    @property
+    def parent(self):
+        return self._config.get(MacVtap.BASE_IFACE)
+
+    @property
+    def _config(self):
```

Comment:
Redefine a internal property is not good.  Just change that to public interface as `config_subtree`.

Reply:
Done. Thank you!